main/picc/picc.h

```diff
@@ -0,0 +1,66 @@
+#ifndef _HEADER_PICC_
+#define _HEADER_PICC_
+
+#include "../prelude.h"
+
+/**
+ * The maximum length of the network SSID
+ */
+#define PICC_SSID_LENGTH 32
+/**
+ * The maximum length of the network password
+ */
+#define PICC_PASS_LENGTH 43
+/**
+ * The maximum length of the website URL
+ */
+#define PICC_SID_LENGTH SENSOR_ID_LENGTH
```

Comment:
Waarom niet geset?

Reply:
It does, it just refers to another define